src/core/infrastructure/KyRequester.ts

```diff
@@ -26,7 +25,7 @@ function defaultRequest(service: any, { body, query, sudo, method }) {
   if (sudo) headers.append('sudo', `${sudo}`);
 
   if (typeof body === 'object' && !(body instanceof FormData)) {
-    bod = JSON.stringify(decamelizeKeys(body, skipAllCaps));
+    bod = JSON.stringify(decamelizeKeys(body));
```

Comment:
Does it avoid decamelizing if the input is in the form of "SOME_UPPER_CASE_VARIABLE" ?

Reply:
![Screenshot_20191022-050808](https://user-images.githubusercontent.com/827205/67256975-fdf68800-f489-11e9-8e36-3b8b8cae592b.jpg)